docs/website/docs/user-guides/v3-migration-guide.md

```diff
@@ -0,0 +1,131 @@
+      targetPort: 3000
+    - name: "5858-tcp"
+      targetPort: 5858
+```
+
+### Commands added, modified or removed in v3
+
+The following table contains a list of odo commands that have either been modified or removed. In case of a 
+modification, the modified command is mentioned in the `v3` column. Please refer below legend beforehand:
+* 👷 currently not implemented, but might get implemented in future
+* ❌ not implemented, no plans for implementation
+
+| v2           | v3 |
+|--------------|--|
+| app delete   | ❌ |
+| app describe | ❌ |
+| app list     | ❌ |
+| catalog describe service| ❌ |
+| catalog describe component | registry --details |
+| catalog list service | 👷odo list services |
+| catalog list component | registry |
+| catalog search service | ❌ |
+| catalog search component | registry --filter |
+| config set | ❌ |
+| config unset | ❌ |
+| config view | ❌ | 
+| debug info | ❌ (not needed as debug mode is start with odo dev --debug command that blocks terminal, the application can be running in inner-loop mode only if odo dev is running in terminal) |
+| debug port-forward  | ❌ (port forwarding is automatic when users execute `odo dev --debug` as long as [the endpoint is defined in the devfile](#changes-to-the-way-component-debugging-works)) |
+| env set  | ❌ |
+|env uset | ❌ |
+| env view | ❌ |
+|preference set| preference set|
+|preference unset| preference unset|
+|preference view| preference view|
+|project create| create namespace|
+|project delete| delete namespace|
+|project get| ❌ |
+|project list| list namespace|
+|project set| set namespace|
+|registry add| preference add registry|
+|registry delete| preference remove registry|
+|registry list| preference view |
+| registry update|no command for update. If needed, it can be done using preference remove registry and preference add registry|
+|service create/delete/describe/list|❌|
+|storage create/delete/list|❌|
+|test|👷(will be implemented after v3-GA) #6070|
+|url create/delete/list| ❌ (odo dev automatically sets port forwarding between container and localhost. If users for some reason require Ingress or Route for inner-loop development they will have to explicitly define them in the devfile as kubernetes components)|
+|build-images|build-images| 
+|deploy|deploy|
+|login|login|
+|logout|logout|
+|create / component create|init|
+|delete / component delete|delete component|
+|describe / component describe|describe component|
+|exec / component exec|❌|
+|link / component link|add binding|list / component list|
+|list|log / component log|
+|logs|push / component push|in v3 there is only dev command that behaves more like watch in v2. There is an option to disable automatic reloads when a file is changed (--no-watch) in future there will be an option to trigger “sync” explicitly when --no-watch option is used)|
+|status / component status|❌|
+|unlink / component unlink|remove binding|
+|watch / component watch|dev|
+|❌|describe binding|
+|❌|list binding|
+|❌|analyze |
```

Comment:
Format this table? https://tabletomarkdown.com/format-markdown-table/

Reply:
Done.